### PR TITLE
New Gas StETH motions

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: src/governance/easy-track/abi/EvmAllowedRecipientsRegistryStETH.abi.json
Comment (dgusakov) [inline src/governance/easy-track/abi/EvmAllowedRecipientsRegistryStETH.abi.json:1]: Looks like duplicate

File: src/governance/easy-track/abi/EvmAddGasSupplyAllowedRecipientStETH.abi.json
Comment (vgorkavenko) [inline src/governance/easy-track/abi/EvmAddGasSupplyAllowedRecipientStETH.abi.json:1]: Could you make it a one-line file?

**Review by madlabman — APPROVED** [review]

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: src/governance/easy-track/abi/EvmAllowedRecipientsRegistryStETH.abi.json
Comment (skhomuti) [reply src/governance/easy-track/abi/EvmAllowedRecipientsRegistryStETH.abi.json:1]: many abis here are duplicates but from different contracts (instances), it's not a problem in my view

File: src/governance/easy-track/abi/EvmAddGasSupplyAllowedRecipientStETH.abi.json
Comment (skhomuti) [reply src/governance/easy-track/abi/EvmAddGasSupplyAllowedRecipientStETH.abi.json:1]: linter force it

Comment (vgorkavenko) [reply src/governance/easy-track/abi/EvmAddGasSupplyAllowedRecipientStETH.abi.json:1]: Could we add their to ignore list?

Comment (skhomuti) [reply src/governance/easy-track/abi/EvmAddGasSupplyAllowedRecipientStETH.abi.json:1]: fixed

**Review by madlabman — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Pull request merged**